networkit/cpp/io/DibapGraphReader.cpp

```diff
@@ -161,15 +161,15 @@ Graph DibapGraphReader::read(const std::string &path) {
         for (index v = 0; v < (count)V; ++v) {
             for (index e = of[v]; e < (index)of[v + 1]; ++e) {
                 if (v <= (index)to[e]) {
-                    graph.addEdge(v, to[e], ew[e]);
+                    graph.addEdge(v, to[e], ew[e], true);
```

Comment:
This (and in other files below) changes the behavior of the readers right? Since we do not support multi-edges, I think it is ok to discard them by default. However, we should properly document this in the notes of the next release.